addons/website/views/snippets/s_faq_horizontal.xml

```diff
@@ -0,0 +1,98 @@
+                            <p><br/>The first step in the onboarding process is <b>account creation</b>. This involves signing up on our platform using your email address or social media accounts. Once you’ve created an account, you will receive a confirmation email with a link to activate your account. Upon activation, you’ll be prompted to complete your profile, which includes setting up your preferences, adding any necessary payment information, and selecting the initial features or modules you wish to use.</p>
+                            <p>Next, you will be introduced to our <b>setup wizard</b>, which is designed to guide you through the basic configuration of the platform. The wizard will help you configure essential settings such as language, time zone, and notifications.</p>
+                            <p><a href="#">Read More <i class="fa fa-angle-right" role="img"/></a></p>
+                        </span>
+                    </article>
+                </div>
+                <div data-name="Topic" class="s_faq_horizontal_entry col-12 mb-2 pt16 pb32">
+                    <article class="row">
+                        <hgroup class="col-lg-4" role="heading">
+                            <div class="s_faq_horizontal_entry_title position-lg-sticky pb-lg-3 transition-base overflow-auto" style="top: 16px">
+                                <h3 class="h5-fs">Updates and Improvements</h3>
+                                <p class="o_small text-muted">We are committed to continuous improvement, regularly releasing updates and new features based on user feedback and technological advancements.</p>
+                            </div>
+                        </hgroup>
+                        <span class="col-lg-7 offset-lg-1 d-block">
+                            <p>Our development team works tirelessly to enhance the platform's performance, security, and functionality, ensuring it remains at the cutting edge of innovation.</p>
+                            <p>Each update is thoroughly tested to guarantee compatibility and reliability, and we provide detailed release notes to keep you informed of new features and improvements. </p>
+                            <div data-snippet="s_chart" data-name="Chart" class="s_chart o_draggable" data-type="line" data-legend-position="top" data-tooltip-display="true" data-stacked="false" data-border-width="1" data-data='{"labels":["v15","v16","v17","v18"],"datasets":[{"label":"Improvements","data":["12","24","48","200"],"backgroundColor":"o-color-1","borderColor":"o-color-1"}]}' data-max-value="250" data-ticks-min="NaN" data-ticks-max="250">
+                                <canvas height="0"/>
+                            </div>
+                            <!-- Graph's preview: removed automatically on drop  -->
+                            <img onload="this.remove()" src="website/static/src/img/snippets_previews/s_faq_horizontal_2_preview.jpg" class="w-100"/>
```

Comment:
This is removed as soon as it is loaded - typically it is already removed inside the snippets dialog, before drop.
Wouldn't the `s_dialog_preview` class do the trick instead ?

Reply:
Indeed, I've definitely overthinked it ^^ 
Thanks 👍